### ISSUE TITLE
Hyper 0.14 still used by metrics-exporter-prometheus

### ISSUE THREAD

**Issue by evaneaston** [evaneaston/openweathermap#13]
There's a little bloat in the exporter because it contains two different versions of hyper.

I upgraded the library to use hyper 1.1.  But metrics-exporter-prometheus uses 0.14.

```
openweathermap_exporter v0.2.0 (/home/evan/dev/openweathermap/exporter)
 | ...
├── metrics-exporter-prometheus v0.12.2
│   ...
│   ├── hyper v0.14.26
```

I've opened [a PR](https://github.com/metrics-rs/metrics/pull/450) on metricsrs/metrics and will update the exporter once/if that PR makes it into their crate.

**Comment by evaneaston** [comment]
With [pull #25](https://github.com/evaneaston/openweathermap/pull/25), hyper 1.1 and correct corresponding libraries are used.

```
$ cargo tree | grep hyper
├── hyper v1.1.0
├── hyper-rustls v0.26.0
│   ├── hyper v1.1.0 (*)
│   ├── hyper-util v0.1.3
│   │   ├── hyper v1.1.0 (*)
├── hyper-util v0.1.3 (*)
│   ├── hyper v1.1.0 (*)
│   ├── hyper-tls v0.6.0
│   │   ├── hyper v1.1.0 (*)
│   │   ├── hyper-util v0.1.3 (*)
│   ├── hyper-util v0.1.3 (*)
├── hyper v1.1.0 (*)
├── hyper-util v0.1.3 (*)
```

**Comment by evaneaston** [comment]
Addressed with #25
